app/tests/controllers/console.test.js

```diff
@@ -0,0 +1,21 @@
+describe('Console', function() {
+  'use strict';
+
+  var KeywordObjectCreator;
+
+  beforeEach(module('crate'));
+  beforeEach(function() {
+    angular.mock.inject(function($injector) {
+      KeywordObjectCreator = $injector.get('KeywordObjectCreator');
+    });
+  });
+
+  describe('KeywordObjectCreator', function() {
+    it('should create object from keyword array', inject(function() {
+      var keywordArray = ['select', 'from', 'limit'];
+      var keywordsObject = KeywordObjectCreator.create(keywordArray);
+      expect(keywordsObject).toEqual({'select': true, 'from': true, 'limit': true});
+    }));
+  });
+
```

Comment:
optional: add another test that checks what happens if you have duplicate keywords in the initial array